apps/users/views.py

```diff
@@ -11,6 +13,13 @@
 user_model = get_user_model()
 
 
+class UserTypeSensitivePasswordResetConfirm(PasswordResetConfirmView):
+    def form_valid(self, form):
+        if self.user.organizations.exists():
+            self.success_url = reverse_lazy("orgadmin_password_reset_complete")
+        return super().form_valid(form)
+
+
```

Comment:
I wonder if it would be wise to do a bit of "session invalidation" here. `super()` will handle the django-admin session invalidation. On the SPA side, we potentially have an Authorization cookie with a now-stale JWT. So let's invalidate that cookie--
```suggestion
    def dispatch(self, *args, **kwargs):
        response = super().dispatch(*args, **kwargs)
        # Invalidate Authorization cookie in client browser by setting its value to gibersih and it's max_age to 0
        response.set_cookie(settings.AUTH_COOKIE_KEY, "NoTaVaLiDtOkEn", max_age=0)
        return response
```